projects/optic/src/utils/git-utils.ts

```diff
@@ -49,3 +49,25 @@ export const resolveGitRef = async (ref: string): Promise<string> =>
     const command = `git rev-parse ${ref}`;
     exec(command, cb);
   });
+
+export const findOpenApiSpecsCandidates = async (
+  ref?: string
+): Promise<string[]> =>
+  new Promise((resolve, reject) => {
+    const cb = (err: unknown, stdout: string, stderr: string) => {
+      if (err || stderr || !stdout) reject(err || stderr);
+      resolve(
+        stdout
+          .trim()
+          .split('\n')
+          .filter((path) => !!path)
+      );
+    };
+    const command = `toplevel=$(git rev-parse --show-toplevel) && \
```

Comment:
Extracted this command out from `api add` to be able to use it for diff-all (modifying to grep from a git tree optionally)